open_academy/demo/demo.xml

```diff
@@ -0,0 +1,30 @@
+<odoo>
+    <data>
+<!--
```

Comment:
Same here. Don't add commented code. Please remove this file and all the references to it.

Reply:
This change is pending. Remove this file please and its reference in `__manifest__.py` file.